hub/ui-new/.eslintrc

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
why is this needed as a part of this patch?

Reply:
I did it for ui-new because it helped in solving eslint error for ui-new.